docs/testing/cypress.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Testing with Cypress
+description: Use Cypress to write end-to-end tests with Clerk.
+---
+
+# Testing with Cypress
+
+<Callout type="info">
+  Our `@clerk/testing` package supports only end-to-end testing with Cypress. Unit tests are not supported.
+</Callout>
+
+Cypress is a well-established JavaScript testing framework. This guide aims to help you set up your environment for creating authenticated tests with Clerk. This guide will assume you're somewhat familiar with Clerk and Cypress.
+
+Before diving in, you might want to start by visiting the [Cypress starter](https://github.com/clerk/example-cypress-nextjs) for an example repo that tests a Clerk-powered application using [Testing Tokens](/docs/testing/overview#testing-tokens).
```

Comment:
Let's explain the purpose of this guide better - we want to show users how to incorporate Clerk's testing tokens for their Cypress test. Testing tokens are the star of the show here, so we want to explain them and emphasize their usage.

```suggestion
Testing with Cypress requires the use of Clerk's [Testing Tokens](/docs/testing/overview#testing-tokens) to bypass various bot detection mechanisms that typically safeguard Clerk applications against malicious bots. Without Testing Tokens, you may encounter "Bot traffic detected" errors in your requests.

This guide aims to help you set up your environment for creating Clerk-authenticated tests with Cypress.

> [!NOTE] 
> Check out the [demo repo](https://github.com/clerk/example-cypress-nextjs) that tests a Clerk-powered application using [Testing Tokens](/docs/testing/overview#testing-tokens).
```